common/constants.go

```diff
@@ -12,10 +12,22 @@ const (
 
 var LoadProcess sync.Once
 var version Version
+
+// Set via -ldflags
+// Example (it should all be one line):
+//   go install -ldflags
+//     "-X common.BUILDDATE=`date -u +%d/%m/%Y@%H:%M:%S`
+//      -X common.GITCOMMIT=`git rev-parse HEAD`
+//      -X common.MAJOR=1
+//      -X common.MINOR=2
+//      -X common.PATCH=3"
```

Comment:
To be a proper shell command on multiple lines, each line should end with `\` except the last, no?

Reply:
That's why I said "everything in one line" in the first line of the comment, if I don't split it the linter complains, and I am leery of shell quoting combined with backslashes.